Consensus.md

```diff
@@ -69,15 +79,15 @@ Currency
 The Sia cryptosystem has two types of currency. The first is the Siacoin.
 Siacoins are generated every block and distributed to the miners. These miners
 can then use the siacoins to fund file contracts, or can send the siacoins to
-other parties (presumably in a trade for out-of-band goods and services, or
-simply other currencies.). The siacoin is represented by a 128 bit unsigned
-integer.
+other parties. The siacoin is represented by a 128 bit unsigned integer.
 
 The second currency in the Sia cryptosystem is the Siafund, which is a special
-asset limited to 10,000 indivisible units. Each time a contract is created,
-3.9% of the payout is put into the siafund pool. The number of siacoins in the
-siafund pool must always be divisible by 10,000; the number of coins taken from
-the payout is rounded down to the nearest 10,000.
+asset limited to 10,000 indivisible units. Each time a file contract payout is
+made, 3.9% of the payout is put into the siafund pool. The number of siacoins
+in the siafund pool must always be divisible by 10,000; the number of coins
+taken from the payout is rounded down to the nearest 10,000. The siafund is
+also represented by a 128 bit unsigned integer, even though only 16 bits of
+resolution are required.
```

Comment:
well, it *was* so we could just use the same Currency type we already had, but...

my comment on #316, I was thinking we should probably just use a big.Int everywhere and have no fixed amount of upwards precision.... \*ducks\*